docs/userguide/storagedriver/imagesandcontainers.md

```diff
@@ -116,17 +172,21 @@ One way to do this is using a Dockerfile and the `docker build` command.
         Removing intermediate container 2023460815df
         Successfully built 03b964f68d06
 
-    > **Note:** The period (.) at the end of the above command is important. It tells the `docker build` command to use the current working directory as its build context.
+    > **Note:** The period (.) at the end of the above command is important. It
+    >  tells the `docker build` command to use the current working directory as
+    >   its build context.
 
     The output above shows a new image with image ID `03b964f68d06`.
 
-3. Run the `docker images` command to verify the new image is in the Docker host's local storage area.
+5. Run the `docker images` command to verify the new "changed-ubuntu" image is 
+6. in the Docker host's local storage area.
 
         REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
         changed-ubuntu      latest              03b964f68d06        33 seconds ago      131.4 MB
-        ubuntu  
+        ubuntu  			latest				013f3d01d247		6 weeks ago			131.3 MB
```

Comment:
careful; looks like  you're using tabs here, but the lines above use spaces. We might have slightly reduced the space between columns to make them better fit in the documentation